firmware/integration/rusefi_config.txt

```diff
@@ -482,19 +487,19 @@ MAP_sensor_config_s map;@see isMapAveragingEnabled
 ThermistorConf clt;todo: merge with channel settings, use full-scale Thermistor here!
 ThermistorConf iat;
 	
-	int launchRpm;A secondary Rev limit engaged by the driver to help launch the vehicle faster;"rpm", 1, 0, 0, 20000, 2
+	int launchRpm;+A secondary Rev limit engaged by the driver to help launch the vehicle faster;"rpm", 1, 0, 0, 20000, 2
 	int launchTimingRetard;;"deg", 1, 0, -180, 180, 2
 	int hip9011PrescalerAndSDO;+value '6' for 8MHz hw osc\nread hip9011 datasheet for details\ntodo split into two bit fields;"integer", 1, 0, 0, 32, 0
 	float knockBandCustom;+We calculate knock band based of cylinderBore\n Use this to override - kHz knock band override;"kHz", 1, 0, 0, 20, 2
 
 
-float[DWELL_CURVE_SIZE] sparkDwellRpmBins;On Single Coil or Wasted Spark setups you have to lower dwell at high RPM;"RPM", 1, 0, 0, 18000, 2
+float[DWELL_CURVE_SIZE] sparkDwellRpmBins;+On Single Coil or Wasted Spark setups you have to lower dwell at high RPM;"RPM", 1, 0, 0, 18000, 2
 	float[DWELL_CURVE_SIZE] sparkDwellValues;;"ms", 1, 0, 0, 30, 2
 
 struct_no_prefix specs_s
-float displacement;Engine displacement, in litres\nsee also cylindersCount;"L", 1, 0, 0, 1000, 2
+float displacement;+Engine volume/capacity, in litres\nsee also cylindersCount;"L", 1, 0, 0, 1000, 2
```

Comment:
what was wrong with `displacement`?

Reply:
Using displacement as a way to explain displacement kind of fails the brief. 
Think looking in the dictionary to see: Displacement - The displacement of an engine.

Reply:
![image](https://user-images.githubusercontent.com/48498823/140189624-6bd57df5-c6d4-4582-aa37-111d94c5b205.png)

@OrchardPerformance please confirm that you see tooltip
@OrchardPerformance please confirm that you do NOT see '+' in master version of this file